benchflow-dsl/src/main/scala/cloud/benchflow/dsl/explorationspace/ExplorationSpaceGenerator.scala

```diff
@@ -0,0 +1,126 @@
+package cloud.benchflow.dsl.explorationspace
+
+import cloud.benchflow.dsl.definition.BenchFlowTest
+import cloud.benchflow.dsl.definition.types.bytes.Bytes
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com)
+ *         created on 2017-05-25
+ */
+object ExplorationSpaceGenerator {
+
+  type ServiceName = String
+  type VariableName = String
+  type VariableValue = String
+  type NumUsers = Int
```

Comment:
this is just a way in Scala to specify types to make the code more easy to read, basically instead of `Int` I can use `NumUsers` to make it more clear what type a list is etc.